olad/RDMHTTPModule.cpp

```diff
@@ -1163,6 +1181,16 @@ void RDMHTTPModule::SupportedSectionsDeviceInfoHandler(
       case ola::rdm::PID_CURVE:
         AddSection(&sections, CURVE_SECTION, CURVE_SECTION_NAME);
         break;
+      case ola::rdm::PID_DIMMER_INFO:
+        AddSection(&sections, DIMMER_INFO_SECTION, DIMMER_INFO_SECTION_NAME);
+        break;
+      case ola::rdm::PID_MINIMUM_LEVEL:
+        AddSection(&sections, DIMMER_MINIMUM_SECTION,
+                   DIMMER_MINIMUM_SECTION_NAME);
+        break;
+      case ola::rdm::PID_MAXIMUM_LEVEL:
+        AddSection(&sections, DIMMER_MAXIMUM_SECTION,
+                   DIMMER_MAXIMUM_SECTION_NAME);
     }
```

Comment:
Add a final break; at the end.
```suggestion
         break;
     }
```

Reply:
Done